src/media/media.service.ts

```diff
@@ -93,7 +102,16 @@ export class MediaService {
     return url;
   }
 
-  public async deleteFile(filename: string, username: string): Promise<void> {
+  /**
+   * @async
+   * Try to delete the file specified by the filename with the user specified by the username.
+   * @param {string} filename - the name of the file to delete.
+   * @param {string} username - the username of the user who uploaded this file
+   * @return {string} the url of the saved file
+   * @throws {PermissionError} the user is not permitted to delete this file.
+   * @throws {NotInDBError} - the file entry specified is not in the database
+   */
+  async deleteFile(filename: string, username: string): Promise<void> {
```

Comment:
I think we should remove the username parameter from this and check in the controller if the user has the permission to do that. Therefore it would be good to have an `isOwner` function in the permission service which has a `mediaUpload` object as parameter. Should we make another PR for that or use #777 ?

Reply:
that is correct. I'd suggest doing that in a different PR.

Reply:
ok